tests/integration/test_utils_vcs_git.py

```diff
@@ -110,6 +110,10 @@ def remote_default_ref(_remote_refs: FetchPackResult) -> bytes:
 def remote_default_branch(remote_default_ref: bytes) -> str:
     return remote_default_ref.decode("utf-8").replace("refs/heads/", "")
 
+def test_use_system_git_client_from_environment_variables():
```

Comment:
Let's add a docstring linking back to the original issue and labeling this a regression test (we test the primary functionality of env var -> config else where and it will help explain why this orphan test exists).